doc/configuration.md

```diff
@@ -233,6 +233,109 @@ var (
 )
 ```
 
+### allow-cuddle-used-in-block
+
+Controls if you may cuddle variables used in block statements. This is false by default as to not change existing behavior.
```

Comment:
After this is merged the part about not changing behavior will no longer make much sense so we can just drop it. Also the fault is shown right under. Maybe clarify to say it's anywhere within the immediate following block?

```suggestion
Controls if you may cuddle variables used anywhere in the following block.
```